### DISCUSSION TITLE
Display molten-nvim kernel in status line

### DISCUSSION THREAD

**Discussion by Bain-OS** [NvChad/NvChad#2965] [General]
Hey, thanks for making this project!

Sorry if this is a stupid question, I am new to neovim + lua 

In molten you must first init a kernel, then you can run python against that kernel. The molten docs [here](https://github.com/benlubas/molten-nvim/tree/main?tab=readme-ov-file#status-line) show that I can use `require('molten.status').kernels()` to get the currently active kernel

The screenshot in this issue demonstrates what I'm after - https://github.com/benlubas/molten-nvim/issues/210 - in the bottom right `python310`.

How might I be able to do this?

Cheers

**Comment by siduck** [comment]
u need to override statusline modules. read from here https://nvchad.com/docs/config/nvchad_ui/#tabufline_and_statusline

```lua
modules = {
    molt = function ()
        return " molten : " ..require('molten.status').kernels()  
    end
}
```
example! make sure to add it in ur order too
